src/composables/transactionToken.ts

```diff
@@ -102,11 +118,73 @@ export function useTransactionToken({
     },
   );
 
-  const isDex = computed(() => transaction.value
+  const isDex = computed<boolean>(() => !!(transaction.value
     && getDexContracts.value && transaction.value.tx.contractId && (
     getDexContracts.value.router.includes(transaction.value.tx.contractId)
     || getDexContracts.value.wae.includes(transaction.value.tx.contractId)
-  ));
+  )));
+
+  const labels = computed<TranslateResult[]>(() => {
```

Comment:
If the type `TranslateResult[]` is put in square brackets after the `computed` it means that we are just guessing what will be returned. So this code for example will be correct:
```
const a = computed<number>(() => 'lorem ipsum');
```

In order to properly type this I suggest to type the returned function. This way the returned body will also by validated:
```
computed((): (string | TranslateResult)[] => ...
```